api/handler.py

```diff
@@ -0,0 +1,70 @@
+import json
+import os
+from api import get_co2_estimate
+from transport_co2 import Mode
+
+def strtofloat(val):
+    if isinstance(val, str):
+        val = float(val)
+    return val
+
+def post(event, context):
+    try:
+        vars = event.get("queryStringParameters", dict())
+        if vars != None:
+            print(f'estimate-co2 sourceIp={event["requestContext"]["identity"]["sourceIp"]} userAgent={event["requestContext"]["identity"]["userAgent"]}, input={vars}')
```

Comment:
Should we use the Python logging framework here?

Reply:
done